backend/composer/models.py

```diff
@@ -240,18 +241,66 @@ class Meta:
         verbose_name_plural = "Anatomical Entities"
 
 
-class Layer(AnatomicalEntityMeta):
-    ...
+class Layer(models.Model):
+    layer_id = models.BigAutoField(primary_key=True, auto_created=True)
+    ae_meta = models.ForeignKey(AnatomicalEntityMeta, on_delete=models.CASCADE, related_name='layer_meta', null=True)
```

Comment:
issue (blocking): null should be set to False. A layer should **always** be associated with an anatomical entity meta